src/test/kotlin/net/revanced/patcher/ReaderTest.kt

```diff
@@ -1,12 +1,13 @@
 package net.revanced.patcher
 
+import com.sun.xml.internal.messaging.saaj.util.ByteOutputStream
 import kotlin.test.Test
 
 internal class ReaderTest {
     @Test
     fun `read jar containing multiple classes`() {
-        val testData = PatcherTest::class.java.getResourceAsStream("/test2.jar")!!
-        Patcher(testData, PatcherTest.testSigs) // reusing test sigs from PatcherTest
-        testData.close()
+        val testData = PatcherTest::class.java.getResourceAsStream("/test1.jar")!!
```

Comment:
```suggestion
        val testData = PatcherTest::class.java.getResourceAsStream("/test2.jar")!!
```
`test1.jar` only has 1 class. This test is to test if the implementation of `Io` can read multiple classes from a JAR file.
`test2.jar` actually has 2 classes, so this one needs to be used for this test.